lib/etcdctl.py

```diff
@@ -90,12 +91,14 @@ def member_list(self, leader_address=False):
             unit_name = unit.split(" ")[1].split("=")[-1]
             peer_urls = unit.split(" ")[2].split("=")[-1]
             client_urls = unit.split(" ")[3].split("=")[-1]
+            is_leader = unit.split(" ")[4].split("=")[-1]
```

Comment:
the above sample text doesn't show this "leader" field.  Has the output changed from a different etcd version?  Does this still support the prior etcd versions?